integration/self_hosted/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    materialize = {
+      source = "materialize.com/devex/materialize"
+    }
+  }
+}
+
+provider "materialize" {
+  host     = "materialized"
+  port     = 6877
+  database = "materialize"
```

Comment:
Missed this one.

Reply:
This is only for the internal integration tests which are using the Docker emulator. It allows me to test some resources that the `materialize` user does not have the rights to, like setting system parameters for example:

```hcl
resource "materialize_system_parameter" "default_cluster" {
  name  = "max_tables"
  value = "2000"
}
```

This will not be exposed to the end users and the orchestratord.